[AV-104690] 8.0 on Capella - FTS Document Filters (#393)
couchbaselabs/docs-devex: modules/search/pages/create-type-mapping.adoc

. xref:type-mapping-options.adoc#collection[Configure your type mapping options].
. Click btn:[Add To Index].

[#filter-mapping]
==== Add a Document Filter to a Collection Type Mapping

You can add a xref:customize-index.adoc#type-identifiers[document filter] to any collection type mapping on your Search index.

> Document filters add an additional filter to any documents included in your Search index for a collection type mapping.

Reviewer: ```suggestion
Document filters allow you to add filters to any documents included in your Search Index for a collection type mapping.
```
